lib/widgets/product_listing.dart

```diff
@@ -455,7 +455,12 @@ class _ProductListingState extends State<ProductListing> {
       color: paletteDarkGreyColor,
       height: constants.imageHeight,
       width: 500,
-      child: _getProductListingImage(product, pageController),
+      child: Stack(
```

Comment:
@DavidCuentasMar this change has been made in the ProductListing widget so, it will show up in both Admin and Customer views. Can you please add a condition such that the _getProductStockQuantityTag(product) will be added to the stack only if the current signed user is admin. Something like:

```
List<Widget> _widgetList = [
  _getProductListingImage(product, pageController),
];

if (currentSignedInUser.isAdmin()) {
  _widgetList.add(_getProductStockQuantityTag(product));
}

Stack(
  children: _widgetList
)
```